cvat/apps/engine/migrations/0062_delete_previews.py

```diff
@@ -0,0 +1,60 @@
+import os
+import logging
+import sys
+import traceback
+
+from django.db import migrations
+from django.conf import settings
+
+def delete_previews(apps, schema_editor):
+    migration_name = os.path.splitext(os.path.basename(__file__))[0]
+    migration_log_file = '{}.log'.format(migration_name)
+    stdout = sys.stdout
+    stderr = sys.stderr
+    # redirect all stdout to the file
+    log_file_object = open(os.path.join(settings.MIGRATIONS_LOGS_ROOT, migration_log_file), 'w')
+    sys.stdout = log_file_object
+    sys.stderr = log_file_object
+
+    log = logging.getLogger(migration_name)
+    log.addHandler(logging.StreamHandler(stdout))
+    log.addHandler(logging.StreamHandler(log_file_object))
+    log.setLevel(logging.INFO)
+
+    def delete_object_previews(db_objects, root_path):
+        for db_obj in db_objects:
+            preview_path = os.path.join(root_path, str(db_obj.id), 'preview.jpeg')
+            try:
+                os.remove(preview_path)
+            except Exception as e:
+                log.error(f'Cannot delete path {preview_path}')
+                log.error(str(e))
+                traceback.print_exc(file=sys.stderr)
+
+    log.info('\nDeleting Data previews...')
+    Data = apps.get_model('engine', 'Data')
+    delete_object_previews(Data.objects.all(), settings.MEDIA_DATA_ROOT)
+
+    log.info('\nDeleting Job previews...')
+    Job = apps.get_model('engine', 'Job')
+    delete_object_previews(Job.objects.all(), settings.JOBS_ROOT)
```

Comment:
Potentially we can have millions of jobs. The line can be a problem from the performance point of view. On one side it is the right approach. On another side, if we just remove all `{settings.JOBS_ROOT}/{num}/preview.jpeg` files will it be a problem?

Reply:
>>> def f():
...   now =datetime.now()
...   l = list(Job.objects.all())
...   print(datetime.now() - now)
...   print(len(l))
...
>>> f()
0:00:09.785674
1015453